asmrun/spacetime_snapshot.c

```diff
@@ -259,6 +259,8 @@ static value take_snapshot(double time_override, int use_time_override)
   CAMLassert(sizeof(double) == sizeof(value));
   v_time = allocate_outside_heap_with_tag(sizeof(double), Double_tag);
   Double_field(v_time, 0) = time;
+  /* CR doligez: either this will have to change for no-flat-float-array,
+     or the ML-side type will change to floatarray */
```

Comment:
These are just supposed to be floats, not float arrays -- I think I may have used the wrong macro here.  Could you fix it?  (I think you can remove the CRs then.)